js/data/siteHacks.js

```diff
@@ -11,6 +11,26 @@ const googleTagManagerRedirect = 'data:application/javascript;base64,' + base64E
 
 module.exports = {
+  'sp1.nypost.com': {
+    enableForAdblock: true,
+    enableForTrackingProtection: true,
+    onBeforeRequest: function(details) {
+      return {
+        redirectURL: 'data:application/javascript;base64,MA=='
+      }
+    }
+  },
+  'nypost.com': {
+    enableForAdblock: true,
+    enableForTrackingProtection: true,
+    onBeforeRequest: function(details) {
+      if (urlParse(details.url).pathname !== '/content-control.html') {
```

Comment:
This one should just move to the abp-filter-parser-cpp data list, I'll just make a note to remove it here when I'm working on cosmetic filterds.